src/Core/Application/Service/ProjectSettingsHandler.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Service;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\Repository\SettingRepositoryInterface;
+use SprykerSdk\Sdk\Core\Application\Dto\ProjectSettingsInitDto;
+use SprykerSdk\Sdk\Infrastructure\Service\Setting\ProjectSettingsInitializer\ProjectSettingsInitializer;
+
+class ProjectSettingsHandler
```

Comment:
It must have an interface, 'cause it uses in the other layers. Also, `Handler` is a forbidden name in spryker.

Also, the class is not only `handles` logic, but also contains some business logic in the `writeProjectSettings`. As the easiest solution, I suggest renaming it to the `ProjectSettingInitializer` and renaming the current `ProjectSettingsInitializer` to the `ProjectSettingsInitializerProcessor`.

Reply:
No tests were provided.